app/models/approval.rb

```diff
@@ -6,10 +6,8 @@ class Approval < ActiveRecord::Base
     end
     state :actionable do
       event :approve, transitions_to: :approved
-      event :reject, transitions_to: :rejected
     end
     state :approved
-    state :rejected
```

Comment:
Removing `rejected` made this code easier, but you're right, a separate PR would be preferable and will cut down the size of this PR.